docs/root/start/_include/envoy-dynamic-demo.yaml

```diff
@@ -0,0 +1,49 @@
+node:
+  cluster: test-cluster
+  id: test-id
+
+dynamic_resources:
```

Comment:
For a dynamic config quick start would it be better to use filesystem xDS? This could actually work out of the box and we could guide the user around updating the file system contents for different things? WYDT? My concern about this is that the control plane doesn't actually point anywhere so this isn't going to do much as a quick start? (I know you have a sandbox coming that actually works, but maybe we do filesystem here and then point to the sandbox for something with go-control-plane)

Reply:
hmm - ok

i agree that it would be better if the example actually worked without a lot of extra setup

i wasnt sure about filesystem xds, which is why ive focused more on control plane, but yep agreed it would be an easier start

if we go ahead this way can i suggest i do the following

- remove all the dynamic stuff from this PR - and just keep the other cleanups
- (quickly) create a filesystem xds sandbox - so im confident of how it works and we have a workthrough example
- i create another PR with config explanations (like i have done here) but for fsxds and/or control plane

Reply:
we could separate out the pages also in that PR

Reply:
Sure SGTM